openedx_webhooks/jira/tasks/update_latest_github_activity.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```

Comment:
1. In this case `github` in the name refers to JIRA fields we're using to track GitHub activities.
2. Separation of concerns. This way we know that this set of tasks *only* interact with JIRA.